### DISCUSSION TITLE
博客加载出现问题

### DISCUSSION THREAD

**Discussion by yk555** [volantis-x/hexo-theme-volantis#788] [Q&A]
博客时，导航条和头像下方的github等图标加载缓慢，并且F12查看，出现CORS报错

**Comment by yk555** [comment]
![image](https://user-images.githubusercontent.com/55798994/180416163-56cff759-88af-4ef4-99a4-e71f9d996a38.png)

**Comment by yk555** [comment]
![image](https://user-images.githubusercontent.com/55798994/180416234-1288431e-8b8d-41c0-b47a-a86ac17d3d41.png)

**Comment by Tikas** [comment]
这是你那里的区域无法访问 cdn 导致的，如果是自己的博客，建议把字库换个 cdn 或者直接放服务器里读取
